include/Communication/BTSerialCommunicationManager.h

```diff
@@ -15,6 +15,11 @@
 #include <Ws2bth.h>
 #include <BluetoothAPIs.h>
 
+#ifdef _WIN32
+	#pragma comment(lib, "Ws2_32.lib")
+	#pragma comment(lib, "Bthprops.lib")
+#endif
```

Comment:
Let's keep this in for now, but this is strictly a msvc compiler option. It would be a good idea to put this in the cmake instead.